packages/@orchidui-vue/src/DataDisplay/ListItem/components/OcTerminal.vue

```diff
@@ -22,8 +26,11 @@ defineProps({
     class="p-4 flex gap-x-4 rounded items-center border border-gray-200 group"
     :class="{ 'hover:shadow-normal': !isDisabled }"
   >
-    <div class="w-[56px] shrink-0 aspect-square bg-oc-bg-dark rounded">
-      <img :src="image" alt="terminal" />
+    <div
+      class="shrink-0 aspect-square bg-oc-bg-dark rounded overflow-hidden"
+      :class="`w-[${imageSize}px]`"
```

Comment:
I think it will not work because tailwind does not work on dynamic binding
https://tailwindcss.com/docs/content-configuration#dynamic-class-names